applications/nrf_desktop/README.rst

```diff
@@ -284,6 +284,41 @@ To use the nRF Desktop project with your custom board:
    Some options have dependencies that might not be needed when these options are disabled.
    For example, when the LEDs module is disabled, the PWM driver is not needed.
 
+FLASH Memory layout
+===================
+
+Depending on whether the bootloader was enabled, partition layout on FLASH memory is set either by:
+
+* Defining layout in DTS files.
+* Defining layout in partition manager configuration.
+
+Set of required partitions differs depending on a configuration.
+There must be at least one partition where code is stored and one partition to store settings.
+The bootloader, if enabled, will add additional partitions to the set.
+
+Memory layout in DTS
```

Comment:
I think it would be good to explain that not the whole layout described in DTS is in use

Reply:
DTS defines: boot partition, 2 slot partitions and even scratch partition

Reply:
I have added the explanation.
When I was at it I thought that it would be better to enable use of DTS code partitions.

Reply:
Maybe we could ask for making partition manager available also for configurations without bootloader?
This would simplify everything a lot.